rpcs3/Emu/Cell/PPUCallback.h

```diff
@@ -171,9 +171,11 @@ namespace ppu_cb_detail
 		FORCE_INLINE static void call(ppu_thread& CPU, u32 pc, u32 rtoc, T... args)
 		{
 			const bool stack = _bind_func_args<0, 0, 0, T...>(CPU, args...);
+			const u64 old_r1 =  CPU.gpr[1];
 			CPU.gpr[1] -= stack ? FIXED_STACK_FRAME_SIZE : 0x70; // create reserved area
+			CPU.gpr[1] &= -16; // Ensure 16-byte alignment
```

Comment:
It must be fixed before binding args, otherwise it shifts arg displacement.

Reply:
Fixed.